_docs/quick-start.md

```diff
@@ -1,25 +1,175 @@
 ---
 layout: single
-title: "Welcome to AutoFixture documentation site"
+title: "Quick Start"
 classes: wide
 ---
 
-This is intro doc.
+## Installing AutoFixture
 
-And here is the basic usage example:
+To add AutoFixture to your project you'll need to install the package from NuGet, using either the command line or the package manager in your IDE.
 
-```c#
+Here are some examples on how you can install the AutoFixture NuGet package.
+
+### .NET CLI
+
+```sh
+> dotnet add package AutoFixture --version 4.13.0
+```
+
+### Package Manager
```

Comment:
I would instead show how to do that via csproj file. Somehow I think nobody uses PM these days :)